src/test/java/com/cqse/teamscaleupload/NativeImageIT.java

```diff
@@ -88,13 +93,21 @@ public void insufficientPermissions() {
     }
 
     @Test
-    public void successfulUpload() {
+    public void successfulSingleFormatUpload() {
         ProcessUtils.ProcessResult result = runUploader(new Arguments());
         assertThat(result.exitCode)
                 .describedAs("Stderr and stdout: " + result.stdoutAndStdErr)
                 .isZero();
     }
 
+    @Test
+    public void successfulSessionUpload() {
```

Comment:
if I understand correctly, the single format case also uses a session, so maybe rename to `successfulMultiFormatUpload`?